modules/wallpapers/ReadMe.txt

```diff
@@ -15,6 +15,7 @@ ReactOS_CirclesBlue_2.jpg           | See above
 ReactOS_CirclesGold_1.jpg           | See above                                      | See above
 ReactOS_CirclesGold_2.jpg           | See above                                      | See above
 ReactOS_Hexagons.jpg                | See above                                      | http://www.reactos.org/forum/viewtopic.php?p=109866#p109866
+Silhouette collection				| Jake Collins (polar@ever3st.com)				 | https://ever3st.com/Graphics/Loot/
```

Comment:
Please use spaces instead of tabs here. That change was ok in previous PR.

Also why don't you visit our chat? We could help you with git. :slightly_smiling_face:

Reply:
which chat? and changing now